docusaurus-docs/community/contribute/local-setup-ui.md

```diff
@@ -9,46 +9,78 @@ To get started with conda-store-ui development, there are a couple of options. T
 
 ## Pre-requisites
```

Comment:
I might update this section in my pull request to point to the page that outlines the git cloning & branching workflow: https://github.com/pavithraes/conda-store/blob/contrib-docs/docusaurus-docs/community/contribute/contribute-code.md#setup-for-local-development

Reply:
That almost makes sense to me, but I'm not sure where/how Docker fits into this suggestion. Should we move the Docker installation instructions out of the core, ui, and juptyerlab-extension docs and into the contribute-code doc?

Reply:
I think we can keep Docker here, and only remove the git & GitHub specific instructions in favour of the above doc

Reply:
Sounds good 👍